moonraker/components/paneldue.py

```diff
@@ -5,6 +5,7 @@
 # This file may be distributed under the terms of the GNU GPLv3 license.
 
 from __future__ import annotations
+from xmlrpc.client import boolean
```

Comment:
I believe that this is an unintentional import, likely pulled in from an annotation below.